src/LtiOidcLogin.php

```diff
@@ -90,7 +90,8 @@ public function doOidcLoginRedirect($launch_url, array $request = null)
             $auth_params['lti_message_hint'] = $request['lti_message_hint'];
         }
 
-        $auth_login_return_url = $registration->getAuthLoginUrl().'?'.http_build_query($auth_params, '', '&');
+        $separator = count(explode('?', $registration->getAuthLoginUrl())) > 1 ? '&' : '?';
```

Comment:
```suggestion
        if (parse_url($registration->getAuthLoginUrl(), PHP_URL_QUERY)) {
            $separator = '&';
        } else {
            $separator = '?';
        }
```
I think this might be a more accurate way to do this.

Also, this is a purely subject stylistic decision, but we try to avoid doing `$var = $condition ? $val1 : $val2;` in favor of the more verbose syntax in my suggestion above.

Reply:
Yep, agreed. I personally prefer a single line ternary in cases like this, but it is very much a personal/project preference. Updated accordingly.